src/Util/Lock.php

```diff
@@ -18,30 +18,30 @@
  */
 class Lock {
        /**
-         * @brief Check for memcache and open a connection if configured
-         *
-         * @return object|boolean The memcache object - or "false" if not successful
-         */
-        public static function memcache() {
-                if (!function_exists('memcache_connect')) {
-                        return false;
-                }
+	 * @brief Check for memcache and open a connection if configured
+	 *
+	 * @return object|boolean The memcache object - or "false" if not successful
+	 */
+	private static function connect_memcache() {
```

Comment:
Please use camelCase format for the method name (`private static function connectMemcache`)